test/unit/property-base.test.js

```diff
@@ -1,6 +1,8 @@
 var _ = require('lodash'),
     expect = require('expect.js'),
-    PropertyBase = require('../../lib/index.js').PropertyBase;
+    Sdk = require('../../lib/index.js'),
```

Comment:
We only capitalize class names. Since `Sdk` is not a classname, use lowercase. (Also, if you've done this in the other PR for `.parent`, fix it in this one.